assertion_jwt_grant_type.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2015 Rodrigo Rafael Monti Kochenburger
```

Comment:
So this is actually not part of Doximity officially, it's a lib that I maintain. Personally I don't really care, but this has always been the license for this repo.